### ISSUE TITLE
Doesn't seem to be obeying the refresh_time variable...

### ISSUE THREAD

**Issue by noblesin** [Ataraksia/NeweggBot#17]
Has anyone else had this problem? It's constantly trying to add the Item to Cart (attempts at less than a second) when I have the refresh_time set to '10' for 10 seconds. This had my IP get blocked from NewEgg. I can't seem to find where in the code it's causing it to do that or if there is something I'm missing or mis-configured.

**Comment by jabramsja** [comment]
I don't think it's just this script that's getting your IP banned. It looks like a new thing that Newegg implemented yesterday to stop bots. There may have to be a variable refresh rate or something so that IP's don't get blocked.

**Comment by Frosty8738** [comment]
@noblesin Its working on Codolophier's fork

**Comment by chundel** [comment]
> I don't think it's just this script that's getting your IP banned. It looks like a new thing that Newegg implemented yesterday to stop bots. There may have to be a variable refresh rate or something so that IP's don't get blocked.

Yup lots of people complaining on nowinstock about getting banned using browser refresher extension too. looks like bans are 12 hours or so

**Comment by shahaf600** [comment]
It's an infinite while loop that trying to add the item to cart, I just added a sleep there
